gulpfile.js

```diff
@@ -120,12 +137,46 @@ function webpackComponents(done) {
 		done();
 	})
 }
+
 function webpackServices(done) {
 	var webpack_config = require('./build/webpack/webpack.services.js')
 	webpack(webpack_config, function (err, stats) {
 		done();
 	})
 }
+
+function angularBuild(done) {
+	try {
+		var baseCmd = 'ng build --base-href "/components/'
```

Comment:
I don't like splitting the quotes like this. Makes it hard to read. Might be better to do something like:

```js
const outputPath = path.join(__dirname, row.source, row["output-directory"]);
const command = `ng build --base-href "/components/${compName}/" -- outputPath "${outputPath}"`;
```

Reply:
noted